config.go

```diff
@@ -237,6 +237,11 @@ type Config struct {
 
 	// skipStartup allows NewRaft() to bypass all background work goroutines
 	skipStartup bool
+
+	// NoLegacyTelemetry allow to skip the legacy metrics to avoid duplicates.
+	// legacy metrics are which has `_peer_name` as metric suffix instead as labels.
+	// e.g: raft_replication_heartbeat_peer0
```

Comment:
```suggestion
	// NoLegacyTelemetry allows to skip the legacy metrics to avoid duplicates.
	// legacy metrics are those that have `_peer_name` as metric suffix instead 
	// of labels.
	// e.g: raft_replication_heartbeat_peer0
```